src/opengeodeweb_back/geode_functions.py

```diff
@@ -13,43 +13,47 @@
 import werkzeug
 
 # Local application imports
-from .geode_objects import objects_list
+from .geode_objects import geode_objects_dict
 
 
-def get_input_factory(geode_object: str):
-    return objects_list()[geode_object]["input_factory"]
+def geode_object_value(geode_object: str):
```

Comment:
What is the goal of this? What is the added value?

Reply:
This is mainly for easier testing